### PR TITLE
[PLAY-15] Made background kit work with CSS Modules

### PR DESCRIPTION
**Pull request by CooperSawyer1**
#### Screens

Screen Shot 2021-11-17 at 10 18 20 AM

Screen Shot 2021-11-17 at 10 20 03 AM
``
#### Breaking Changes

No breaking changes. I moved the classname prop to the end of the tag in the background kit and deleted the lazyload because there is no image here. 

#### Runway Ticket URL

`https://nitro.powerhrg.com/runway/backlog_items/PLAY-15`

#### How to test this

There is no way to test this in playbook. it just works 

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.

- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

**Comment by jasoncypret** [comment]
@CooperSawyer1 you've got conflicts.

```
git checkout master
git pull
git checkout [yourbranch]
git merge origin/master
git push --force-with-lease
```

You can accept all incoming if you get conflicts

File: playbook/dist/playbook.css [hunk 1]
```diff
@@ -2906,9 +2906,12 @@ Background colors ------------------*/
     [class^=pb_badge_kit].dark[class*=_primary] {
       background: rgba(1, 78, 190, 0.2); }
 
+<<<<<<< HEAD
```
Comment (jasoncypret) [inline playbook/dist/playbook.css:2909]: You've got merge conflicts here. You can reset this single file to master by running `git checkout origin/master playbook/dist/playbook.css` then push up that change

File: playbook/app/pb_kits/playbook/pb_background/docs/style.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+.classy {
+  border: 1rem solid green;
+}
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_background/docs/style.scss:4]: Are you wanting to keep this in production?

File: playbook/app/pb_kits/playbook/pb_background/docs/_background_white.jsx [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
 import { Background } from '../..'
+import styles from './style.css'
 
 const BackgroundWhite = (props) => (
   <div>
     <Background
         backgroundColor="white"
         padding="xl"
         {...props}
+        className={styles.classy}
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_background/docs/_background_white.jsx:11]: Do we want to keep these in production?

File: playbook/app/pb_kits/playbook/pb_background/docs/_background_light.jsx [hunk 4]
```diff
@@ -1,11 +1,13 @@
 import React from 'react'
 import { Background } from '../..'
+import styles from './style.scss'
 
 const BackgroundLight = (props) => (
   <Background
       backgroundColor="light"
       padding="xl"
       {...props}
+      className={styles.classy}
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_background/docs/_background_light.jsx:10]: Do you want to keep these in production...or just for testing?

File: playbook/app/pb_kits/playbook/pb_background/_background.scss [hunk 5]
```diff
@@ -30,6 +30,8 @@ $background_colors: (
     filter: blur(0);
   }
 
-
+  // .Classy {
+  //   border: 1rem solid green;
+  // }
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_background/_background.scss:35]: Lets delete instead of comment out

**Review by jasoncypret — CHANGES_REQUESTED** [review]

**Comment by jasoncypret** [comment]
@CooperSawyer1 Note to the change to the title bracket.

File: playbook/dist/playbook.css (hunk 1 above)
Comment (CooperSawyer1) [reply playbook/dist/playbook.css:2909]: will do

File: playbook/app/pb_kits/playbook/pb_background/docs/style.scss (hunk 2 above)
Comment (CooperSawyer1) [reply playbook/app/pb_kits/playbook/pb_background/docs/style.scss:4]: No, I can delete this.

File: playbook/app/pb_kits/playbook/pb_background/docs/_background_white.jsx (hunk 3 above)
Comment (CooperSawyer1) [reply playbook/app/pb_kits/playbook/pb_background/docs/_background_white.jsx:11]: No, I can delete this

File: playbook/app/pb_kits/playbook/pb_background/docs/_background_light.jsx (hunk 4 above)
Comment (CooperSawyer1) [reply playbook/app/pb_kits/playbook/pb_background/docs/_background_light.jsx:10]: These were just for testing. I will delete

File: playbook/app/pb_kits/playbook/pb_background/_background.scss (hunk 5 above)
Comment (CooperSawyer1) [reply playbook/app/pb_kits/playbook/pb_background/_background.scss:35]: will do

**Review by jasoncypret — APPROVED** [review]

**Comment by jasoncypret** [comment]
LGTM 🚀

**Comment by RobGentile17** [comment]
First PR lets gooooo 🥶

**Pull request merged**